defaults.go

```diff
@@ -63,6 +63,19 @@ func coalesce(v ...interface{}) interface{} {
 	return nil
 }
 
+// fromJson decodes JSON into a structured value, ignoring errors.
+func fromJson(v string) interface{} {
+	output, _ := mustFromJson(v)
+	return output
+}
+
+// mustFromJson decodes JSON into a structured value, returning errors.
+func mustFromJson(v string) (interface{}, error) {
+	var output interface{}
+	err := json.Unmarshal([]byte(v), &output)
```

Comment:
This will silently fail if `v` cannot be converted to a `[]byte`. The type conversion should be checked.

Reply:
That's a compile time check. I believe all strings can be converted to byte slices. (This is not a type assertion.)

Reply:
Sorry, you're right. However, I would extend the function to also accept `[]byte` as an input.

Reply:
Why?

Reply:
Data flowing between template functions can be of any type. It's entirely possible that someone will want to use this function with a `[]byte`, for example:

    {{ (readFile "foo.json" | fromJson).bar }}

where `readFile` returns a `[]byte`. Why not allow this?

Reply:
We can. I was just wondering what the use case was.